docs/quick-tutorials/k8s-network-policies.mdx

```diff
@@ -236,8 +236,8 @@ If you've attached Otterize OSS to Otterize Cloud, go back to see the [access gr
 
 It's now clear what happened:
 1. The server is now protected, and is also blocking some of its clients. Click on it to see what to do about it.
-2. Calls from **[client]** are declared and therefore allowed (solid green line).
-3. Calls from **[client-other]** are not declared and therefore blocked (dashed red line). Click on the line to see what to do about it.
+2. Calls from **[client]** are declared and therefore allowed (green line).
+3. Calls from **[client-other]** are not declared and therefore would be blocked (yellow line). Click on the line to see what to do about it.
```

Comment:
This tutorial instructs you to remove shadow mode and doesn't cover these concepts, so if you install with cloud it says to "omit the following flag in the Helm command: --set intentsOperator.operator.mode=defaultShadow", which would make the lines red.

@sapirwo Can you update the screenshots to reflect this? The screenshots should be as in the original ones as the behavior should not have changed, the changes should just be the new style (no dashed, new nodes)

Reply:
👍